Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -368,17 +368,19 @@ public function queryFilterMultiple($data, $clauseType = 'must')
      *
      * Access all aggregation data with {nodes.aggregations} in your fluid template
      *
-     * @param $name
-     * @param $field
-     * @param string $type
-     * @param null $parentPath
+     * @param string $name The name to identify the resulting aggregation
+     * @param string $field The field to aggregate by
+     * @param string $type Aggregation type
+     * @param string $parentPath
+     * @param int $size The amount of buckets to return
      * @return $this
      */
-    public function fieldBasedAggregation($name, $field, $type = "terms", $parentPath = null)
+    public function fieldBasedAggregation($name, $field, $type = 'terms', $parentPath = '', $size = 10)
```

Comment:
Lets start here. Adding the $size as last argument means that I have to give the parentPath in eel. (do we have `null` in eel?). 
If we have that - As the path is a string, a default value of null is completely unintuitive for integrators and  a '' given in eel would not work.

Thats why I changed the API here. I am open to your suggestions.